src/radical/pilot/configs/resource_anl.json

```diff
@@ -16,6 +16,7 @@
         "agent_scheduler"             : "CONTINUOUS",
         "agent_spawner"               : "POPEN",
         "launch_methods"              : {
+                                         "order": ["APRUN"],
```

Comment:
What happens if that is not specified - should't the launch order fall back to `['APRUN']` anyway?

Reply:
Thanks, makes sense.